extension/src/fileSystem/index.test.ts

```diff
@@ -177,15 +177,25 @@ describe('getModifiedTime', () => {
 describe('findOrCreateDvcYamlFile', () => {
   it('should make sure a dvc.yaml file exists', () => {
     const cwd = '/cwd'
-    void findOrCreateDvcYamlFile(cwd, '/my/training/script.py', 'train')
+    void findOrCreateDvcYamlFile(
+      cwd,
+      '/my/training/script.py',
+      'train',
+      scriptCommand.PYTHON
+    )
 
     expect(mockedEnsureFileSync).toHaveBeenCalledWith(`${cwd}/dvc.yaml`)
   })
 
   it('should add the stage name to the dvc.yaml file', () => {
     const cwd = '/cwd'
     const uniqueStageName = 'aWesome_STAGE_name48'
-    findOrCreateDvcYamlFile(cwd, '/script.py', uniqueStageName)
+    findOrCreateDvcYamlFile(
```

Comment:
[Q] Why doesn't this one need `void` but the others do?

Reply:
Missed it during the merge